src/components/hostURL/HostURL.tsx

```diff
@@ -161,7 +161,7 @@ export default class HostURLConfiguration extends Component<HostURLConfigProps,
         return (
             <>
                 <section
-                    className="mt-16 mb-16 ml-20 mr-20 global-configuration__component"
+                    className="global-configuration__component"
```

Comment:
Classname `global-configuration__component` is not making any sense in this component

Reply:
it is existing so didn't change this